edx_proctoring/static/proctoring/js/views/proctored_exam_attempt_view.js

```diff
@@ -0,0 +1,156 @@
+            event.preventDefault();
+        },
+        getPaginatedAttempts: function(event) {
+            var target = $(event.currentTarget);
+            this.collection.url = target.data('target-url');
+            this.hydrate();
+            event.stopPropagation();
+            event.preventDefault();
+        },
+        getCSRFToken: function () {
+            var cookieValue = null;
+            var name = 'csrftoken';
+            if (document.cookie && document.cookie != '') {
+                var cookies = document.cookie.split(';');
+                for (var i = 0; i < cookies.length; i++) {
+                    var cookie = jQuery.trim(cookies[i]);
+                    // Does this cookie string begin with the name we want?
+                    if (cookie.substring(0, name.length + 1) == (name + '=')) {
+                        cookieValue = decodeURIComponent(cookie.substring(name.length + 1));
+                        break;
+                    }
+                }
+            }
+            return cookieValue;
+        },
+        loadTemplateData: function () {
+            var self = this;
+            $.ajax({url: self.tempate_url, dataType: "html"})
+                .error(function (jqXHR, textStatus, errorThrown) {
+
+                })
+                .done(function (template_data) {
+                    self.template = _.template(template_data);
+                    self.hydrate();
+                });
+        },
+        hydrate: function () {
+            /* This function will load the bound collection */
+
+            /* add and remove a class when we do the initial loading */
+            /* we might - at some point - add a visual element to the */
+            /* loading, like a spinner */
+            var self = this;
+            self.collection.fetch({
+                success: function () {
+                    self.render();
+                }
+            });
+        },
+        collectionChanged: function () {
+            this.hydrate();
+        },
+        render: function () {
+            if (this.template !== null) {
+                var self = this;
+                var data = {
+                    proctored_exam_attempts: this.collection.toJSON()[0].proctored_exam_attempts,
+                    pagination_info: this.collection.toJSON()[0].pagination_info,
+                    attempt_url: this.collection.toJSON()[0].attempt_url,
+                    inSearchMode: this.inSearchMode,
+                    searchText: this.searchText
+                };
+                _.extend(data, viewHelper);
+                var html = this.template(data);
+                this.$el.html(html);
+                this.$el.show();
+            }
+        },
+        onRemoveAttempt: function (event) {
+            event.preventDefault();
+
+            // confirm the user's intent
+            if (!confirm(gettext('Are you sure you wish to remove this student\'s exam attempt?'))) {
```

Comment:
Please change to: "Are you sure you ^^want^^ to remove this student's exam attempt?"